Runtime/SharedResources/Scripts/Vrtk4UiToPointer.cs

```diff
@@ -1,139 +1,173 @@
-﻿namespace Tilia.VRTKUI
-{
-    using System;
-    using System.Collections;
-    using System.Collections.Generic;
-    using Tilia.Indicators.ObjectPointers;
-    using UnityEngine;
-    using Zinnia.Cast;
-    using Zinnia.Data.Type;
-    using Zinnia.Pointer;
+﻿using UnityEngine.Serialization;
+using System.Collections.Generic;
+using Tilia.Indicators.ObjectPointers;
+using UnityEngine;
+using Zinnia.Cast;
+using Zinnia.Data.Type;
+using Zinnia.Pointer;
 
+namespace Tilia.VRTKUI
+{
     /// <summary>
     /// Mediator component for working with VRTK 3.3.0 ported UI system
     /// </summary>
     [DisallowMultipleComponent]
     public class Vrtk4UiToPointer : MonoBehaviour
     {
         [SerializeField] private PointerFacade pointerFacade;
-        [SerializeField] private bool applyButtonsFromFacade = false;
-        [SerializeField] private VRTK4_UIPointer UI_Pointer;
-        private List<Vector3> temporalList = new List<Vector3>(2);
-        private ObjectPointer.EventData eventData;
-        private PointsCast.EventData pointerCastEventData;
+        [SerializeField] private bool applyButtonsFromFacade;
+        [SerializeField, FormerlySerializedAs("UI_Pointer")] private VRTK4_UIPointer uiPointer;
+
+        private const float HoverEnterDuration = 0.1f;
+        private const float HoverExitDuration = 0.9f;
+        
+        private readonly List<Vector3> cachedRaycastResultWorldPositions = new(2);
```

Comment:
@ArtemkaKun  
`private readonly List<Vector3> cachedRaycastResultWorldPositions = new List<Vector3>(2);`

Reply:
Ok, got it, will fix in Monday